src/Stratis.SmartContracts.CLR/Loader/ContractAssemblyLoader.cs

```diff
@@ -1,27 +1,47 @@
 ﻿using System;
+using System.IO;
 using System.Reflection;
+using System.Runtime.Loader;
 using CSharpFunctionalExtensions;
 
 namespace Stratis.SmartContracts.CLR.Loader
 {
+    public class SmartContractLoadContext : AssemblyLoadContext
+    {
+        private AssemblyLoadContext defaultContext;
+
+        public SmartContractLoadContext(AssemblyLoadContext defaultContext)
+        {
+            this.defaultContext = defaultContext;
+        }
+
+        protected override Assembly Load(AssemblyName assemblyName)
+        {
+            // Ensure that an exact compatible version is used.
+            if (assemblyName.Name == "Stratis.SmartContracts.Standards" && assemblyName.Version.Major == 1 && assemblyName.Version.Minor < 4)
```

Comment:
As far as i understand these type of changes will cause conflicts between new and old nodes

Reply:
If it is just because of `Stratis.SmartContracts.Standards` so we can include that older interface IStandardToken and rename the new one as IStandard256

Reply:
I think the conflicts will be resolved if we can still provide same IStandardToken interface and implementation as old and add new ones with suffix 256. 

But still this is going to be issue between new and old nodes whenever new contract submited by uint256 and uint128 types.

Reply:
Au contraire! The change is meant to prevent old smart contracts from running with the new incompatible version of the `IStandardToken`  interface. 

The only "issue", which is unrelated to this change, is that old nodes will not understand contracts based off the new datatypes.

Could you perhaps elaborate on what you mean?